src/utils/maintenance/setMEBXPassword.ts

```diff
@@ -1,40 +1,64 @@
+import { HttpHandler } from '../../HttpHandler'
 import { IClientManager } from '../../interfaces/IClientManager'
 import { IConfigurator } from '../../interfaces/IConfigurator'
 import Logger from '../../Logger'
+import { ClientMsg, ClientObject } from '../../models/RCS.Config'
 import { AMTDeviceDTO } from '../../repositories/dto/AmtDeviceDTO'
-import { WSManProcessor } from '../../WSManProcessor'
+import { ClientResponseMsg } from '../ClientResponseMsg'
 import { EnvReader } from '../EnvReader'
 import { MqttProvider } from '../MqttProvider'
-import { RPSError } from '../RPSError'
+import { AMT } from '@open-amt-cloud-toolkit/wsman-messages/index'
+import { parseBody } from '../parseWSManResponseBody'
 
-export const setMEBXPassword = async (clientId: string, message: any, amtwsman: WSManProcessor, clientManager: IClientManager, configurator: IConfigurator): Promise<boolean> => {
-  const logger = new Logger('setMEBXPassword')
+const logger = new Logger('setMEBXPassword')
+const amt = new AMT.AMT()
```

Comment:
I thought we had refactored this to be AMT.Messages()

Reply:
It will be taken care in future PR's once WSMan messages is updated.